### DISCUSSION TITLE
Unable to Configure eligibility policy

### DISCUSSION THREAD

**Discussion by svnaresh** [aws-samples/iam-identity-center-team#282] [General]
I am unable to configure eligibility policy because when i login to TEAM i am not getting  Administration section. We have Active Directory integrated with IDC and i created team admin group in the AD and added my user in this group. In the SSO TEAM Application I gave access to that AD group. So I am able to login to the TEAM UI but not seeing the  Administration section.
Please let me know if any configuration need to modify to see  Administration section.

Thanks in advance.
Venkat Naresh

**Comment by tawoyinfa** [comment]
have you tried logging out and back into the application ?

**Reply by svnaresh** [reply]
Yes, I did multiple times.

**Comment by svnaresh** [comment]
Hi
Actually we are using Active Directory Federated users using SAML integration  instead of direct IAM users. Is this going to work for this TEMP application.
Please let me know as we are still facing the above issue.
Thanks
Naresh

**Reply by svnaresh** [reply]
In the /aws/lambda/teamgetGroups-main cloudwatch logs getting below logs:
GROUP not found. 
user idc_user@ds.example.com removed from Admin group
